README.md

```diff
@@ -4,11 +4,70 @@ Catch common Java mistakes as compile-time errors
 
 [![Build Status](https://travis-ci.org/google/error-prone.svg?branch=master)](https://travis-ci.org/google/error-prone)
 
+## Getting Started
+
 Our documentation for users is at http://errorprone.info
 
-To develop and build error-prone, see our documentation on the [wiki](https://github.com/google/error-prone/wiki/For-Developers).
+### Maven
+
+Add the following configuration to your `pom.xml`.
+
+```xml
+<build>
+  <plugins>
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-compiler-plugin</artifactId>
+      <version>3.1</version>
+      <configuration>
+        <compilerId>javac-with-errorprone</compilerId>
+        <forceJavacCompilerUse>true</forceJavacCompilerUse>
+      </configuration>
+      <dependencies>
+        <dependency>
+          <groupId>org.codehaus.plexus</groupId>
+          <artifactId>plexus-compiler-javac-errorprone</artifactId>
+          <version>2.5</version>
+        </dependency>
+      </dependencies>
+    </plugin>
+  </plugins>
+</build>
+```
+
+### Ant
+
+Download the latest release of
+[error_prone_ant.jar](https://repo1.maven.org/maven2/com/google/errorprone/error_prone_ant)
+from maven, and use the following javac task in your `build.xml`.
+
+```xml
+<javac destdir="build"
+  compiler="com.google.errorprone.ErrorProneAntCompilerAdapter"
+  source="1.7" target="1.7" encoding="UTF-8" debug="true"
```

Comment:
> Do we need this line? Shouldn't source and target default to the host JDK version?

Thanks, I removed the source/target options.

> do you need to tell where to put the file after its downloaded?

It needs to match the `pathelement` in the config. I tried to make that clearer by using `/path/to/error_prone_ant.jar`. I like this ant configuration option because it doesn't rely on `$ANT_HOME` / `~/.ant/lib`. I lifted the example straight from #292.